cmd/kubeadm/app/master/manifests.go

```diff
@@ -367,6 +362,20 @@ func getAPIServerCommand(cfg *kubeadmapi.MasterConfiguration, selfHosted bool) [
 	return command
 }
 
+func getEtcdCommand(cfg *kubeadmapi.MasterConfiguration) []string {
+	var command []string
+
+	defaultArguments := map[string]string{
+		"listen-client-urls":    "http://127.0.0.1:2379",
+		"advertise-client-urls": "http://127.0.0.1:2379",
+	}
+
+	command = append(command, "etcd", "--data-dir=/var/lib/etcd")
```

Comment:
In particular I want to get rid of the etcd dir env param and make the volume mount actually respect what `DataDir` is set to